### PR TITLE
Add back-to-control button for M7 experiment

### PR DESCRIPTION
**Pull request by adamkecskes**
Button added to campaign view allowing user to turn m7 levels off: 
beta-levels-off

### CODE REVIEW

File: app/views/play/CampaignView.coco.coffee [hunk 1]
```diff
@@ -1338,6 +1339,19 @@ module.exports = class CampaignView extends RootView
...
+      { text: 'Yes', onClick: ($noty) =>
+        if me.getM7ExperimentValue() == 'beta'
+          me.updateExperimentValue('m7', 'control')
+          $noty.close()
+          @render()
+      }, { text: 'No', onClick: ($noty) -> $noty.close() }]
+    })
+
+
+
```
Comment (mrfinch) [inline app/views/play/CampaignView.coco.coffee:1353]: remove extra lines

**Review by mrfinch — APPROVED** [review]
I like hover state UI more than the normal one. Do we actually want it to be less prominent?

File: app/views/play/CampaignView.coco.coffee (hunk 1 above)
Comment (adamkecskes) [reply app/views/play/CampaignView.coco.coffee:1353]: yes, it should be ["Something really unobtrusive that most players won't notice unless directed to."](https://app.asana.com/0/0/1202317318282996/1202823842099215/f)
extra lines removed

**Review by nwinter — APPROVED** [review]
It looks good.

Don't need to block merge, but in general I would put confirm "yes/ok/submit" actions on the bottom right of modals and "no/cancel" actions to the left of those (Mac style). This and a couple other of the more recent ones have that reversed (Windows style).

Reasoning? That's the way the rest of the modals on the site do it.

Without needing to be consistent, it might be better to do ok on left, cancel on right, especially if the buttons were aligned to left edge of modals rather than right edge. Or to detect platform (Windows vs. Mac) and adapt to the system default conventions. But we haven't done that, and I think consistency will be better.

More reading: https://ux.stackexchange.com/questions/1072/ok-cancel-on-left-right

**Pull request merged**